### ISSUE TITLE
How to install Jc141/Johncena141 releases ?

### ISSUE THREAD

**Issue by Louisdpb** [Louisdpb/Steam-deck-How-to-#2]
JC141 is a popular repacker that offers a plethora of game releases to the gaming community that runs on Linux . However, it is important to note that his releases are not compatible with the Steam Deck straight out of the box. But fear not, there is a workaround to get the game releases up and running on the Steam Deck.

The method involves extracting the groot.drafts file to gain access to the executable file (.exe) and running it through either Lutris, Heroic Launcher, or even Steam. Here are the steps to follow:

Step 1 - install Distrobox

DistroBox is an open-source software that can be used to create a personal Linux distribution. It allows users to customize and build their operating system image by choosing and installing various packages and software tools available in the official Linux repositori

Follow the [instructions ](https://deckhandheld.com/install-any-linux-distro-on-steam-deck-with-distrobox/)

Step 2 - Download the JC141 Game Release

Next, head over to [JC141](https://1337x.unblockit.asia/johncena141-torrents/2/) and download the game release that you wish to install onto your Steam Deck.

Step 3 - Extract the groot.drafts File

Upon opening your container Fedora in my case, install the following 

`sudo apt-get install dwarfs`

download the game release, locate the groot.drafts file within the folder. Extract this file to gain access to the executable file (.exe) by running the following command.

Step 4 - Add the Game to Lutris or Heroic Launcher

Open up Lutris or Heroic Launcher on your Steam Deck and click on the "Add Game" button. Follow the prompts to add the game release that you just downloaded and extracted.

Step 5 - Run the Game

After adding the game release to Lutris or Heroic Launcher, simply click on the game's icon to run it. The game should launch without any issues.

In conclusion, while JC141 game releases may not have worked straight out of the box on the Steam Deck, with this workaround, you will be able to enjoy a wide variety of games on your portable gaming console.
